### PR TITLE
feat(hub): add kyc banners for india nic

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/acquisition-storage`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-11156
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ [n/a]

## Description

Add a banner for KYC updation when its not yet done

### CODE REVIEW

File: packages/manager/apps/hub/src/dashboard/dashboard.controller.js [hunk 1]
```diff
@@ -14,11 +14,33 @@ export default class DashboardController {
...
+        serviceType: 'aapi',
+      })
+      .then(({ data: featureAvailability }) => {
+        if (featureAvailability['identity-documents']) {
+          this.$http.get(`/me/procedure/identity`).then(({ data }) => {
+            if (['open', 'required'].includes(data.status)) {
+              this.showKycBanner = true;
+            } else {
+              this.showKycBanner = true;
+            }
```
Comment (ghyenne) [inline packages/manager/apps/hub/src/dashboard/dashboard.controller.js:38]: I think there is a mistake
We have twice this.showKycBanner = true;

**Review by ghyenne — CHANGES_REQUESTED** [review]

File: packages/manager/apps/hub/src/dashboard/dashboard.controller.js [hunk 2]
```diff
@@ -14,11 +14,33 @@ export default class DashboardController {
...
+        serviceType: 'aapi',
+      })
+      .then(({ data: featureAvailability }) => {
+        if (featureAvailability['identity-documents']) {
+          this.$http.get(`/me/procedure/identity`).then(({ data }) => {
+            if (['open', 'required'].includes(data.status)) {
+              this.showKycBanner = true;
+            } else {
+              this.showKycBanner = false;
+            }
```
Comment (mohammed-zahaf) [inline packages/manager/apps/hub/src/dashboard/dashboard.controller.js:38]:
```suggestion
this.showKycBanner = ['open', 'required'].includes(data.status)
```

Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/dashboard.controller.js:38]: **nitpick:** can be simplified

```suggestion
              this.showKycBanner = !!['open', 'required'].includes(data.status);
```

File: packages/manager/apps/hub/src/dashboard/dashboard.html [hunk 3]
```diff
@@ -37,6 +37,30 @@
...
+            <a
+                class="oui-link_icon"
+                data-ng-href="{{:: $ctrl.myIdentitySectionLink }}"
+                data-ng-click="$ctrl.trackClick()"
+                target="_blank"
+                rel="noopener"
+            >
+                <span
+                    data-translate="manager_hub_dashboard_kyc_banner_link"
+                    aria-hidden="true"
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/dashboard.html:59]:
```suggestion
```

File: packages/manager/apps/hub/src/dashboard/order-dashboard/template.html [hunk 4]
```diff
@@ -1,4 +1,25 @@
...
+    <a
+        class="oui-link_icon"
+        data-ng-href="{{:: $ctrl.myIdentitySectionLink }}"
+        data-ng-click="$ctrl.trackClick()"
+        target="_blank"
+        rel="noopener"
+    >
+        <span
+            data-translate="manager_hub_dashboard_kyc_banner_link"
+            aria-hidden="true"
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/order-dashboard/template.html:19]:
```suggestion
```

File: packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -8,5 +8,7 @@
   "manager_hub_dashboard_modal_siret_part_2": "Avec l'aide de votre KBIS, cette mise à jour ne prendra que quelques secondes.",
   "manager_hub_dashboard_modal_siret_link": "Mettre à jour",
   "manager_hub_dashboard_modal_siret_cancel": "Pas maintenant",
-  "manager_hub_dashboard_modal_title": "Mise à jour de vos coordonnées"
+  "manager_hub_dashboard_modal_title": "Mise à jour de vos coordonnées",
+  "manager_hub_dashboard_kyc_banner_description": "Vous souhaitez commander un produit OVHcloud: la réglementation en vigueur nécessite une vérification de votre identité. Veuillez ajouter vos documents d'identité.",
```
Comment (mohammed-zahaf) [inline packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json:12]:
```suggestion
  "manager_hub_dashboard_kyc_banner_description": "Vous souhaitez commander un produit OVHcloud : la réglementation en vigueur nécessite une vérification de votre identité. Veuillez ajouter vos documents d'identité.",
```

**Review by mohammed-zahaf — APPROVED** [review]

File: packages/manager/apps/hub/src/dashboard/dashboard.controller.js [hunk 6]
```diff
@@ -14,11 +14,29 @@ export default class DashboardController {
...
+        serviceType: 'aapi',
+      })
+      .then(({ data: featureAvailability }) => {
+        if (featureAvailability['identity-documents']) {
+          this.$http.get(`/me/procedure/identity`).then(({ data }) => {
+            this.showKycBanner = ['open', 'required'].includes(data.status);
+          });
+        } else {
+          this.showKycBanner = false;
+        }
```
[the remark this answers is not in the document]
Comment (sachinrameshn) [reply packages/manager/apps/hub/src/dashboard/dashboard.controller.js]: done

File: packages/manager/apps/hub/src/dashboard/order-dashboard/template.html [hunk 7]
```diff
@@ -1,4 +1,22 @@
 <hub-dashboard-welcome data-name="$ctrl.me.firstname"></hub-dashboard-welcome>
+<!-- Banner for India KYC -->
+<oui-message
+    type="info"
+    on-dismiss="$ctrl.onDismiss()"
+    dismissable
```
[the remark this answers is not in the document]
Comment (y4nnL) [reply packages/manager/apps/hub/src/dashboard/order-dashboard/template.html:6]: Can you please have a second look ? Thanks

**Review by y4nnL — APPROVED** [review]

**Pull request merged**